airbyte-commons-worker/src/main/java/io/airbyte/workers/general/DefaultReplicationWorker.java

```diff
@@ -391,6 +420,22 @@ private static Runnable readFromSrcAndWriteToDstRunnable(final AirbyteSource sou
     };
   }
 
+  private static void acceptSrcControlMessage(final WorkerSourceConfig sourceConfig,
```

Comment:
we can simplify this method signature by passing in the id instead of the entire config

Reply:
this simplification also extends to the various runnable method interfaces e.g ` readFromDstRunnable` doesn't need the `destinationConfig`. Improves readability since it's clearer what scope requires what data.

Reply:
👍🏻 updated in [d6eb05f](https://github.com/airbytehq/airbyte/pull/19811/commits/d6eb05f4a9a4edfee6c566ed888a7711f9c41449)